src/index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
```

Comment:
I think you can remove this file

Reply:
There's no real gain in removing doing so. We can keep it in case we need to test stuff really quickly without having to embed it in OpenPaaS' environment.

Reply:
So can we have the components in it?

Reply:
Yes